docs/components/console/manage-clusters/cluster-capacity.md

```diff
@@ -73,32 +72,32 @@ However, if you find yourself experiencing any of the following, then you might
 
-If your cluster utilization is too high, you can help reduce it by:
+If your cluster load and utilization is too high, you can help reduce it by:
 
 - Scaling down the load. For example, by stopping certain clients. Start with your least critical load, and continue from there.
 - Check your running process instances for known issues that cause high processing, such as:
   - Straight-through processing loops, where there are no wait states. For example, a sub process with an error boundary event which loops back to an activity leading into the sub-process. If you have an activity which consistently throws an error, this will result in a subtle infinite loop where the engine is stuck and cannot process anything else. You would have to cancel this instance, or contact support to force cancel it for you.
 
 :::important
-If your cluster capacity percentage remains high even after attempts to reduce it, you might need to increase your cluster size and scale. See [cluster size](/components/concepts/clusters.md#cluster-size) and [sizing your environment](/components/best-practices/architecture/sizing-your-environment.md).
+If your cluster load percentage remains high even after attempts to reduce it, you might need to increase your cluster size and scale. See [cluster size](/components/concepts/clusters.md#cluster-size) and [sizing your environment](/components/best-practices/architecture/sizing-your-environment.md).
 :::
 
-## How cluster capacity percentage is calculated {#capacity-calculation}
+## How cluster load percentage is calculated {#load-calculation}
 
-Cluster capacity percentage is based on the cluster's [flow control configuration](/self-managed/operational-guides/configure-flow-control/configure-flow-control.md).
+Cluster load percentage is based on the cluster's [flow control configuration](/self-managed/operational-guides/configure-flow-control/configure-flow-control.md).
 
 Essentially, if flow control is configured, every partition is assigned a [write rate](/self-managed/operational-guides/configure-flow-control/configure-flow-control.md#exporting-and-write-rate) and a [write rate limit](/self-managed/operational-guides/configure-flow-control/configure-flow-control.md#write-rate-limit).
 
-This means a partition's cluster capacity is defined as:
+This means a partition's cluster load is defined as:
 
 `(writeRate / writeRateLimit)`
 
-This gives a value between 0 and 1, which is multiplied by 100 to give the partition capacity as a percentage.
+This gives a value between 0 and 1, which is multiplied by 100 to give the partition load as a percentage.
 
-The cluster capacity percentage is then the average of the capacity of all partitions in the cluster.
+The cluster load percentage is then the average of the load of all partitions in the cluster.
 
 :::note
-The partition capacity is _only_ calculated and reported on the current partition leader.
+The partition load is _only_ calculated and reported on the current partition leader.
```

Comment:
Not related to this PR but I missed that earlier - should we say about the 5m delay in reporting over the reality?